cmd/docker/internal/signals/signals_unix.go

```diff
@@ -0,0 +1,14 @@
+//go:build !windows
+// +build !windows
```

Comment:
`!windows` is technically not the same as `unix`... I'd honestly rather buildtag this Unix and provide a compilation error on unknown platforms:
```suggestion
//go:build unix
// +build unix
```